### PR TITLE
Prevent players to throw frag/smoke grenade on build time

### PR DESCRIPTION
**Pull request by DiamondPlane**
I made a new one because https://github.com/MT-CTF/capturetheflag/pull/804#issuecomment-785733553

### CODE REVIEW

File: mods/pvp/grenades/grenades.lua [hunk 1]
```diff
@@ -17,6 +17,12 @@ local fragdef = {
 			return
 		end
 
+        if ctf_match.is_in_build_time() then
+			minetest.get_player_by_name(name):get_inventory():add_item("main", "grenades:frag")
+                                      return
+            minetest.chat_send_player(name, "Match hasn't started yet!")
```
Comment (LoneWolfHT) [inline mods/pvp/grenades/grenades.lua:23]: This is never called because of the return statement above it

File: mods/pvp/grenades/grenades.lua [hunk 2]
```diff
@@ -95,6 +101,14 @@ grenades.register_grenade("grenades:smoke", {
 		local player = minetest.get_player_by_name(pname)
 		if not player or not pos then return end
 
+        if ctf_match.is_in_build_time() then
+            if player then
+            player:get_inventory():add_item("main", "grenades:smoke")
+                                      return
+            minetest.chat_send_player(pname, "Match hasn't started yet!")
```
Comment (LoneWolfHT) [inline mods/pvp/grenades/grenades.lua:108]: This is never called because of the return statement above it

**Review by LoneWolfHT — CHANGES_REQUESTED** [review]
This needs to use tabs instead of spaces

File: mods/pvp/grenades/grenades.lua (hunk 2 above)
Comment (DiamondPlane) [reply mods/pvp/grenades/grenades.lua:108]: Can't edit

**Pull request closed without merging**